<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSelectionForm</class>
 <widget class="QWidget" name="GameSelectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>245</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>245</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="gameSelectlistWidget">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitPushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
